apps/frontend/pages/api/twitch/notification/[...param].ts

```diff
@@ -69,6 +72,26 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
     if (messageType === MessageType.Notification) {
         console.log(req.body.event);
+        const streamStatus = req.body.subscription.type;
+
+        // get the features that are enabled
+        const response = await axios.get(`${env.NEXTAUTH_URL}/api/feature/${param[0]}`);
+        if (response.status === 200) {
+            const featureMapping = response.data as FeatureMapTypes;
+            // conditional code to be called if it is an enable or disable event
+            if (featureMapping.featureMap.tweet) {
+                // call the tweet endpoint
+            }
+            if (featureMapping.featureMap.banner) {
```

Comment:
What I was thinking for these, is maybe we could include the endpoint as a constant or something on the feature itself. So we don't have to write logic per feature. We could just iterate over all the features and call whatever endpoint is specified by the feature. Does that make sense?

Basically every feature is going to be doing this:

```ts
 if (streamStatus === 'stream.online') {
    await axios.post(`${env.NEXTAUTH_URL}/api/<FEATURE>/streamup/${param[0]}`);
}
if (streamStatus === 'stream.offline') {
    await axios.post(`${env.NEXTAUTH_URL}/api/<FEATURE>/streamdown/${param[0]}`);
}
```

We don't have to do that now though.